### DISCUSSION TITLE
Exporting a region

### DISCUSSION THREAD

**Discussion by yunusyurtturk** [pelicanmapping/osgearth#2594] [Support Q&A] [answered]
Hi,
I've been learning osgEarth to achieve 'terrain exporting/terrain building' to a specific format.

For example, geocell N36 W34 will be exported as 3D tiles (imagery, elevation etc. data.)

Is it natively possible with osgEarth? I'm asking because I read something like '...osgEarth is just a runtime rendering SDK, it doesn't have ability to export/save to any specific format' (I will attach the link if I can find the related documentation)

As a summary, my question is is it possible to save some part of the earth as 3D tiles (with elevation, imagery, models, etc. data included)?
Is it supported by default or should I write my own plugin for it?

**Comment by gwaldron** [comment, marked as answer]
No; osgEarth does not write out 3D-Tiles.
